### ISSUE TITLE
[BUG REPORT] ansible crashes v2 beta on Ubuntu 18.04

### ISSUE THREAD

**Issue by davephillips** [Dewb/monome-rack#78]
Ubuntu 18.04 Linux
VCV Rack CE v2.git.219bbaf1

Loading the Ansible module crashes VCV Rack:

```
[43.598 info src/common/FirmwareManager.cpp:180 load] Loading module firmware from /home/dlphilp/src/Rack2/plugins/monome-rack/res/firmware/ansible.so

// ansible //////////////////////////////// 
== FLASH struct size: 227296
first run.
i2c addr: a0Makefile:104: recipe for target 'run' failed
make: *** [run] Segmentation fault (core dumped)

```

Running a debug session:

```
[15.409 info src/common/FirmwareManager.cpp:180 load] Loading module firmware from /home/dlphilp/src/Rack2/plugins/monome-rack/res/firmware/ansible.so

// ansible //////////////////////////////// 
== FLASH struct size: 227296
first run.
i2c addr: a0
Thread 1 "Rack" received signal SIGSEGV, Segmentation fault.
0x00007fffc2d22d89 in initialize_module () at ansible/src/main.c:864
864		clock = &clock_null;
(gdb) 

```

The complete backtrace:

https://pastebin.com/pY8DEVPh

Best regards,

dp

**Comment by Dewb** [comment]
Is this with an Actions-produced build a or a local build?

**Comment by davephillips** [comment]
@Dewb Local build.

**Comment by Dewb** [comment]
This should be fixed, finally, in a53c29c. Build here if you want to test: https://github.com/Dewb/monome-rack/actions/runs/4339366508
